code/modules/research/anomaly/anomaly.dm

```diff
@@ -0,0 +1,53 @@
+// Embedded signaller used in anomalies.
+/obj/item/assembly/signaler/anomaly
```

Comment:
This means an anomaly core can be hit with a screwdriver to make it attachable or not. The examine text also reflects the attachable state. Also, as with other assemblies, stuff can be tacked on.

![image](https://user-images.githubusercontent.com/1496804/104118608-ae200d00-532a-11eb-9d0e-90f0d5aefa13.png)